Dockerfile

```
@@ -0,0 +1,138 @@
+FROM ubuntu:xenial as base
+
+# Warning: This file is experimental.
+
+# Install system requirements
+RUN apt update && \
+    # Global requirements
+    DEBIAN_FRONTEND=noninteractive apt install -y \
```

```suggestion
    DEBIAN_FRONTEND=noninteractive apt-get install --yes \
```
* Prefer `apt-get` over `apt` in scripts (`apt` warns so).
* Prefer long-form flags in scripts.